WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -56,7 +56,7 @@
             <% end %>
           </ul>
         </li>
-        <li class="dropdown">
+        <li class="dropdown <%= params[:controller] == 'persons' || params[:controller] == 'media' ? 'active' : '' %>">
```

Comment:
nit: How about this instead?

```ruby
Set['persons', 'media'].include?(params[:controller])
```

Reply:
That will make everything even cleaner if there are *many* controllers that could set the `active` class. FWIW, the existing Information menu item code uses the `||` approach. Think that should get changed with this code as well?

Reply:
I think it would be better if you can change the already existing code too! :slightly_smiling_face:

Reply:
Ok, made them all the same code structure and added a few more controllers that I was able to find in routes. Reminder that at present I can't run the app locally, so review closely.